packages/notification-center/src/components/notification-center/components/NotificationsListTab.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { IMessage, ChannelCTATypeEnum } from '@novu/shared';
 
 import { useNotifications, useNotificationCenter, useNovuContext, useTranslations } from '../../../hooks';
-import image from '../../../images/no-new-notifications.png';
+import image from '../../../images/no-new-notifications.webp';
```

Comment:
changed images to webp which is supported by all modern browsers: https://caniuse.com/?search=webp